pkg/apiclient/apiclient.go

```diff
@@ -457,15 +457,6 @@ func (c *client) redeemRefreshToken() (string, string, error) {
 	return rawIDToken, refreshToken, nil
 }
 
-// NewClientOrDie creates a new API client from a set of config options, or fails fatally if the new client creation fails.
-func NewClientOrDie(opts *ClientOptions) Client {
-	client, err := NewClient(opts)
-	if err != nil {
-		log.Fatal(err)
-	}
-	return client
-}
-
```

Comment:
I know we don't use this anymore in the CLI, but since it exists in `pkg` i think we may want to keep this for other projects importing argo-cd as a library.

Reply:
Yes exactly, that's why I did not remove this incase someone else is already using it.

Reply:
Wait according to this diff. You _did_ remove this.

Reply:
Oh sorry, I thought I moved the code around in the same pkg. Let me add it back.

Reply:
Reverted it back.